base/applications/clipbrd/lang/id-ID.rc

```diff
@@ -0,0 +1,65 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
```

Comment:
You cannot duplicate LANG_ENGLISH because it already exists. You probably want something like this:
LANGUAGE LANG_INDONESIAN, SUBLANG_DEFAULT

Reply:
sorry